docs/getting_started/index.md

```diff
@@ -41,9 +41,9 @@ Inside your `mysite` folder, run the setup steps necessary for any Django projec
 
 ```sh
 $ pip install -r requirements.txt
-$ ./manage.py migrate
-$ ./manage.py createsuperuser
-$ ./manage.py runserver
+$ python manage.py migrate
+$ python manage.py createsuperuser
+$ python manage.py runserver
```

Comment:
I was surprised by those changes, as this shouldn’t be needed (the file already has a shebang to determine what to run it with). But,

- On our "try Wagtail in 7 lines" on https://wagtail.org/, we do use `python manage.py`, and we’ve researched this a lot – so I assume it’s very likely to work in lots of places
- Django’s tutorial as well uses `python manage.py` (https://docs.djangoproject.com/en/4.1/intro/tutorial01/).

So – great!